frontend/src/bundles/overview/components/goal-widget/goal-widget.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon } from '~/bundles/common/components/components.js';
+
+import { CircularProgress } from './components/components.js';
+
+type WidgetProperties = {
+    title: string;
+    subTitle?: string;
+    value: number;
+    target: number;
+};
+
+const GoalWidget = ({
+    title = 'Track Your Daily Activities',
+    subTitle = '',
+    value = 6,
+    target = 10,
```

Comment:
Its not clear what does 6 and 10 mean so please move it to a constant to avoid magic numbers